runok.js

```diff
@@ -33,6 +33,8 @@ module.exports = {
   async defTypings() {
     console.log('Generate TypeScript definition');
     await npx('jsdoc -c typings/jsdoc.conf.js');
+    await npx('jsdoc -c typings/jsdocPromiseBased.conf.js');
+    fs.renameSync('types.d.ts', 'typings/promiseBasedTypes.d.ts');
```

Comment:
Maybe we should execute after this one?

```
await npx('jsdoc -c typings/jsdoc.conf.js');
```

so types.d.ts to be actually generated for classical setup?

Reply:
Yes, you're right - I will fix it

Reply:
Well, it's confusing, but it was done on purpose.
In fact, the 2nd command generates the file types.d.ts on another location, on the root of the folder (to avoid to overwritten the first one). It's why I rename and move this file.

I will have some comments and make it more consistent.